shell/TestPackage.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = "Stop"
+$LastExitCode = 0
+$env:PATH +=";$($env:SystemDrive)\\ProgramData\\chocolatey\\bin"
+
+# https://github.com/chocolatey/choco/issues/512
+$validExitCodes = @(0, 1605, 1614, 1641, 3010)
+
+$package  = ls c:\packages\*.nupkg
```

Comment:
Then the next one will be wrongly named, the one that takes just the first one :)